src/main/java/com/google/devtools/build/remote/client/RemoteClient.java

```diff
@@ -401,6 +401,17 @@ private static void doRun(RunCommand options, RemoteClient client) throws IOExce
   }
 
   public static void main(String[] args) throws Exception {
+    try {
+      selectAndPerformCommand(args);
+    } catch (io.grpc.StatusRuntimeException e) {
```

Comment:
Nits: import the io.grpc.StatusRuntimeException
a space between if and (e.getMessage(

Also:
maybe compare status code with INTERNAL and do contains "http2" on the message instead? I feel like that would be cleaner.

        Status s = Status.fromThrowable(e);
        if (s.getCode() == Code.INTERNAL && s.getDescription().contains("http2")) {...

Reply:
Thanks for the suggestion, this is indeed cleaner